pages/_app.tsx

```diff
@@ -1,18 +1,276 @@
+    setActiveStep((prevActiveStep) => prevActiveStep + 1);
+  };
+
+  const handleBack = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep - 1);
+  };
+
+  const handleStepChange = (step: number) => {
+    setActiveStep(step);
+  };
 
-export default function App({ Component, pageProps }: AppProps) {
   return (
-    <>
-      <style>
-        {`
-      .sui-result + .sui-result {
-        margin-top: 0px;
-      }
-      `}
-      </style>
-      <Component {...pageProps} />
-    </>
+    <div className="container">
+      {/* <Header /> */}
+      <div
+        className="canvas"
+        style={{
+          width: isSmall ? "90%" : "80%",
+          marginLeft: "auto",
+          marginRight: "auto",
+          paddingBottom: "20px",
+        }}
+      >
+        <Stack spacing={2} direction="row">
+          <Button
+            onClick={() => {}} //todo
+            sx={{ textTransform: "none" }}
+            style={{
+              color: "#000000",
+              fontFamily: "Inter",
+              // fontSize: 28,
+            }} //todo create separate component for this
+          >
+            About Advisory
+          </Button>
+          <Button
+            sx={{ textTransform: "none" }}
+            style={{
+              color: "#000000",
+              fontFamily: "Inter",
+            }}
+          >
+            Mentor Profiles
+          </Button>
+          <Button
+            sx={{ textTransform: "none" }}
+            style={{
+              color: "#000000",
+              fontFamily: "Inter",
+            }}
+          >
+            Apply
+          </Button>
+          <Button
+            sx={{ textTransform: "none" }}
+            style={{
+              color: "#000000",
+              fontFamily: "Inter",
+            }}
+          >
+            FAQ
+          </Button>
+        </Stack>
+        <Box component="main">
+          <img
+            className="header-mentorship-logo"
+            src={advisoryMentorshipLogo}
+            alt="Advisory Mentorship Programme"
+          />
+          <img
+            id="partners"
+            className="header-mentorship-partners"
+            src={advisoryMentorshipPartners}
+            alt="Advisory Mentorship Programme Partners"
+          />
+          <p>
+            The Advisory Mentorship Programme pairs students with working
+            professionals in their fields of interest on a 1-1 basis. Over the
+            course of four months, mentors give an hour each month to meet with
+            their mentee. Over the past 2 years, the programme delivered over
+            8,000 hours of mentorship to 2,211 students with the involvement of
+            1,826 mentors. This year, we’re excited to share that over 1,600
+            working professionals and 27 mentorship partners whose fields of
+            expertise range across 48 different industries have come aboard in
+            support of this programme.
+          </p>
+        </Box>
```

Comment:
This part looks quite similar to the current `Header` component, so I just used that instead.